hudi-common/src/main/java/org/apache/hudi/common/config/DFSPropertiesConfiguration.java

```diff
@@ -128,14 +127,19 @@ public void addPropsFromFile(Path filePath) {
         filePath.toString(),
         Option.ofNullable(hadoopConfig).orElseGet(Configuration::new)
     );
+    try {
+      if (isDefaultDirectory && !fs.exists(filePath)) {
```

Comment:
I think we still need to check if the props file exists if the user configures it.  It doesn't have to be an error if the file cannot be found.